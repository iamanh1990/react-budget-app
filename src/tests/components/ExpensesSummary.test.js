import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render correctly with 1 expense', () => {
	const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={231} />);
	expect(wrapper).toMatchSnapshot();
});

test('should render correctly with multiple expenses', () => {
	const wrapper = shallow(<ExpensesSummary expensesCount={23} expensesTotal={2310144} />);
	expect(wrapper).toMatchSnapshot();
});
