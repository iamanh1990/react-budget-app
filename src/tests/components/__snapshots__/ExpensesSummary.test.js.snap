// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with 1 expense 1`] = `
<div>
  Viewing 1 expense totaling $ 2.31
</div>
`;

exports[`should render correctly with multiple expenses 1`] = `
<div>
  Viewing 23 expenses totaling $ 23,101.44
</div>
`;
